.travis.yml

```diff
@@ -39,6 +39,11 @@ script:
   - sudo make -C mtf/metadata check
   - sudo make -C examples/testing-module check-mtf-metadata
   - sudo make -C examples/testing-module check-openshift
-  #- sudo make -C examples/testing-module check-mtf-pdc-module-info-reader
+  - sudo make -C examples/testing-module check-mtf-pdc-module-info-reader
+  - sudo make -C examples/s2i-python-sample check
+  - sudo make -C examples/s2i-python-sample check-python-3.5
```

Comment:
I would prefer to call one make for s2i-python-sample

Reply:
I'll do that.